README.md

```diff
@@ -126,47 +88,84 @@ kustomize build 'github.com/kubernetes-sigs/descheduler/kubernetes/deployment?re
 
 See the [user guide](docs/user-guide.md) in the `/docs` directory.
 
-## Policy and Strategies
+## Policy, Default Evictor and Strategy plugins
+
+**⚠️ v1alpha1 configuration is still suported, but deprecated (and soon will be removed). Please consider migrating to v1alpha2 (described bellow). For previous v1alpha1 documentation go to docs/deprecated/v1alpha1.md**
+
+The Descheduler Policy is configurable and includes default eviction strategy plugins that can be enabled or disabled. It includes a common eviction configuration at the top level, as well as configuration from the Evictor plugin (Default Evictor, if not specified otherwise). Top-level configuration and Evictor plugin configuration are applied to all evictions.
 
-Descheduler's policy is configurable and includes strategies that can be enabled or disabled. By default, all strategies are enabled.
+### Top Level configuration
+
+These are top level keys in the Descheduler Policy that you can use to configure all evictions.
+
+| Name | Default Value | Description |
```

Comment:
Let's add another `Type` column saying type of a param. E.g. string, []string, int, ...

Reply:
:heavy_check_mark: